### PR TITLE
[docs] Add instructions for debugging native runtime code on Android

### PR DESCRIPTION
**Pull request by lambdageek**

### CODE REVIEW

File: docs/workflow/debugging/mono/android-debugging.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Install Android Studio.
+
+Download the symbols nupkg corresponding to the runtime pack that is used by
+the app.  The runtime pack for the android workload is in a folder like
+`${DOTNET_ROOT}/packs/Microsoft.NETCore.App.Runtime.Mono.android-x86/6.0.0-rc.1.21451.13`
+
+The symbols are in a package called
+`Microsoft.NETCore.App.Runtime.Mono.android-x86.6.0.0-rc.1.21451.13.symbols.nupkg`
+uploaded to (FIXME: where does the symbols nuget go?).  Extract it to some folder using `unzip` and in the
```
Comment (lambdageek) [inline docs/workflow/debugging/mono/android-debugging.md:37]: I don't know how to do this without digging through AzDO pipeline control panels.

Is there a nice method to download a symbols .nupkg for a given runtime pack? /cc @akoeplinger @steveisok

Comment (steveisok) [reply docs/workflow/debugging/mono/android-debugging.md:37]: /cc @directhex

File: docs/workflow/debugging/mono/android-debugging.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+5. In the "Path Mappings" section, select the toplevel folder and add a Local
+   Path to a git checkout of the `release/6.0` tree corresponding to the nuget.  Click Apply Changes.
+6. Start an emulator or connect to a device
+7. On the menu bar select  "Run > Edit Configurations..." and on the "Debugger" tab make sure the "Debug Type" is something other than "Java Only"
+8. Start debugging.
+9. You should now have function names, local variables, as well as stepping through the runtime C code.
+
+Since you're debugging an optimized release build, it is likely the debugger will not be able to materialize every local variable.
+
+## Native dbeugging using a local debug build of Mono
```
Comment (thaystg) [inline docs/workflow/debugging/mono/android-debugging.md:56]: typo

**Comment by thaystg** [comment]
We could also add information about how to debug any issue in the debugger?
Usually I run this on adb: `adb shell setprop debug.mono.extra "debug=10.0.2.2:5000,loglevel=10,timeout=1000000000000000"`
And start sdb and start listening on 127.0.0.1 5000
The other steps are similar to yours.

**Review by lateralusX — APPROVED** [review]
LGTM!

**Review by thaystg — APPROVED** [review]

**Pull request merged**